Eplant/tutorial.md

```diff
@@ -0,0 +1,227 @@
+# Tutorial <!-- omit in toc -->
+
+- [Main Component](#main-component)
+- [Eplant Layout](#eplant-layout)
+  - [useViewData](#useviewdata)
+- [View Interface](#view-interface)
+  - [getInitialState:](#getinitialstate)
+  - [getInitialData:](#getinitialdata)
+  - [reducer:](#reducer)
+  - [actions:](#actions)
+  - [component:](#component)
+  - [icon:](#icon)
+  - [name:](#name)
+  - [id:](#id)
+  - [description:](#description)
+  - [thumbnail:](#thumbnail)
+  - [citation:](#citation)
+  - [header:](#header)
+- [state](#state)
+  - [Atoms:](#atoms)
+  - [Hooks:](#hooks)
+  - [Simplicity:](#simplicity)
+  - [Reactivity:](#reactivity)
+- [When you create a new `view`](#when-you-create-a-new-view)
+  - [Define GetStartedView Object:](#define-getstartedview-object)
+  - [Define GetStartedView Component:](#define-getstartedview-component)
+  - [Define Tile Component:](#define-tile-component)
+  - [Summary](#summary)
+- [Eplant2](#eplant2)
+
+
+
+## Main Component
+In the root(`Eplant/main.tsx`), you can see the `Eplant` component, whchi is our main component of our application.  
+
+`main.tsx`
+```
+<React.StrictMode>
+  <Provider>
+    <BrowserRouter>
+      <Config.Provider value={defaultConfig}>
+        <Eplant />
+      </Config.Provider>
+    </BrowserRouter>
+  </Provider>
+</React.StrictMode>
+```
+
+We only have one single Route at the moment but we will have one Route for one component.  
```

Comment:
I suggest that we clarify here that it is the `Eplant.tsx` that handles routing. But you are correct, eventually we will expect to have one Route for each component.